### PR TITLE
Tag normalization

### PR DESCRIPTION
**Pull request by dabcoder**
Addresses https://github.com/DataDog/datadogpy/issues/485. Manually tested too.

### CODE REVIEW

File: datadog/util/format.py [hunk 1]
```diff
@@ -20,3 +20,11 @@ def force_to_epoch_seconds(epoch_sec_or_dt):
     if isinstance(epoch_sec_or_dt, datetime.datetime):
         return calendar.timegm(epoch_sec_or_dt.timetuple())
     return epoch_sec_or_dt
+
+
+def normalize_tags(tag_list):
+    final_tag_list = []
+    for tag in tag_list:
+        # remove commas and whitespaces from the tag
+        final_tag_list.append(tag.replace(",", "_").replace(" ", ""))
```
Comment (zippolyte) [inline datadog/util/format.py]: I would do the minimum client side, and let the API deal with everything. i.e. I would not touch the whitespace if not needed, just replace the comma

Comment (dabcoder) [reply datadog/util/format.py]: Fair enough, will do thanks.

File: datadog/util/format.py [hunk 2]
```diff
@@ -20,3 +20,11 @@ def force_to_epoch_seconds(epoch_sec_or_dt):
...
+    return final_tag_list
```
Comment (ofek) [inline datadog/util/format.py]: For performance, please

```python
def normalize_tags(tag_list):
    return [tag.replace(',', '_') for tag in tag_list]
```

**Comment by dabcoder** [comment]
@jd Actually I looked at it a bit more and it is possible to add a parameter `constant_tags`: https://github.com/DataDog/datadogpy/blob/80c9603381187e43a900e60c8ef4ac9ff019a857/datadog/dogstatsd/base.py#L60-L61 (A list of strings). Unsure if we'd need to make the same change there though.

**Pull request merged**